src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -780,3 +730,8 @@ static void fetch_servers_callback(http_response_t* response)
     window_invalidate_by_class(WC_SERVER_LIST);
 }
 #endif
+
+static bool is_version_valid(const std::string &version)
+{
+    return version.empty() || version == NETWORK_STREAM_ID;
```

Comment:
Hm, why are empty versions valid?

Reply:
When I first implemented this, I must have said that an empty version was a wildcard or "unchecked".